test/decoder_test.exs

```diff
@@ -64,38 +65,6 @@ defmodule MozartFetcher.DecoderTest do
 
       assert Decoder.decode_config(json, %Config{}) == expected
     end
-
-    test "when a map in the list of JSON has an invalid key, the other components are still returned" do
```

Comment:
instead of removing this test, do we want to test the new behaviour when a component is invalid?

Reply:
Ahh yes I forgot about that, was too eager to check it worked aha